indexer-app/src/cmr/indexer/data/concepts/collection/platform.clj

```diff
@@ -23,23 +23,22 @@
    not present."
   [kms-index short-name]
   (let [full-platform
-        (merge default-platform-values
-               (kms-lookup/lookup-by-short-name kms-index :platforms short-name))
+        (kms-lookup/lookup-by-short-name kms-index :platforms short-name)
         {:keys [basis category sub-category short-name long-name uuid]
          ;; Use the short-name from KMS if present, otherwise use the metadata short-name
          :or {short-name short-name}} full-platform]
     {:basis basis
-     :basis-lowercase (str/lower-case basis)
+     :basis-lowercase (when basis(string/lower-case basis))
```

Comment:
You can use the `cmr.common.util/safe-lowercase` function here to simplify the code.

Reply:
I have updated the code.